src/main/flight/mixer.c

```diff
@@ -690,6 +690,20 @@ static void applyFlipOverAfterCrashModeToMotors(void)
     }
 }
 
+static void applyConstantIdleModeToMotors(void)
+{
+    if (ARMING_FLAG(ARMED)) {
+        for (int i = 0; i < motorCount; ++i) {
+            motor[i] = motorOutputLow;
+        }
+    } else {
+        // Disarmed mode
+        for (int i = 0; i < motorCount; i++) {
+            motor[i] = motor_disarmed[i];
```

Comment:
I would probably rewrite it using a ternary operator, as this loop is likely to be unrolled and would require more flash space the way it is now.